deployment/helm/templates/udev.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   protocol:
     udev:
       udevRules:
-      {{- toYaml .Values.udev.udevRules | nindent 6 }}
+      {{- required "Please set at least one udev rule with `--set udev.udevRules[0]==\"<udev rule\"' to specify what you want discovered. See the udev Configuration document for more information." .Values.udev.udevRules | toYaml | nindent 6 }}
```

Comment:
Should we have a link to the configuration udev doc in the error message?

Reply:
Like this entire link: https://github.com/deislabs/akri/blob/main/docs/udev-configuration.md ?